lib/pages/new_private_chat/widget/expansion_list.dart

```diff
@@ -240,6 +192,31 @@ class _ExpansionList extends State<ExpansionList> {
       ),
     );
   }
+
+  Widget get newGroupButton => _IconTextTileButton(
+        context: context,
+        onPressed: () => widget.newPrivateChatController.goToNewGroupChat(),
+        iconData: Icons.supervisor_account_outlined,
+        text: L10n.of(context)!.newGroupChat,
+      );
+
+  Widget get _getHelpsButton => _IconTextTileButton(
+        context: context,
+        onPressed: () => {},
+        iconData: Icons.question_mark,
+        text: L10n.of(context)!.getHelp,
+      );
+
+  Widget get _moreListTile => Padding(
+        padding: const EdgeInsets.symmetric(horizontal: 8.0, vertical: 12.0),
+        child: Text(
+          L10n.of(context)!.more,
+          style: Theme.of(context).textTheme.labelLarge?.copyWith(
+                letterSpacing: 0.1,
+                color: LinagoraRefColors.material().neutral[40],
+              ),
+        ),
+      );
```

Comment:
Extract a widget as a method it is considered to be a pretty bad pattern. We should use `stateless or stateful`

Reply:
done